core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1096,7 +1150,7 @@ paths:
                 type: array
                 items:
                   type: object
-                  description: workspaceNamespace/workspaceName -> bucket migration metadata
+                  description: workspaceNamespace/workspaceName -> bucket migration metadata. bucket migration metadata is an empty list for workspaces that have not been scheduled for migration before
```

Comment:
Nit on al of these-- I would transform into completely sentences or at least remove the period. For instance:

workspaceNamespace/workspaceName -> bucket migration metadata (which will be an empty list for workspaces that have never been scheduled for migration)